integration-tests/cli/discount-code-importer.it.js

```diff
@@ -47,6 +48,10 @@ describe('DiscountCode tests', () => {
           createHttpMiddleware({ host: apiConfig.apiUrl }),
         ],
       })
+
+      return clearData(apiConfig, 'discountCodes')
```

Comment:
okay... You should also clear cart discounts after the discount codes. That is more of an issue